### PR TITLE
Add stack and arguments to panel.

### PR DESCRIPTION
**Pull request by jonathanKingston**
Fix checking the correct feature name.
Move checks into DDG proxy to simplify methods.

**Reviewer:**

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: shared/js/content-scope/utils.js [hunk 1]
```diff
@@ -130,19 +133,43 @@ export function defineProperty (object, propertyName, descriptor) {
...
+        return letter.toUpperCase()
+    })
+}
+
 export class DDGProxy {
-    constructor (objectScope, property, proxyObject) {
+    constructor (featureName, objectScope, property, proxyObject) {
         this.objectScope = objectScope
         this.property = property
+        this.featureName = featureName
```
Comment (sammacbeth) [inline shared/js/content-scope/utils.js:146]: `featureName`'s only use seems to be for `postDebugMessage` where it has to be camelCased. Why not just set it in camelCase originally?

**Review by sammacbeth — COMMENTED** [review]
Looks good. Just wondering why we don't just use camelCase for `featureName` from the start?

File: shared/js/content-scope/utils.js (hunk 1 above)
Comment (jonathanKingston) [reply shared/js/content-scope/utils.js:146]: `shouldExemptMethod` works on the raw name instead which works just on the passed raw names.

Comment (jonathanKingston) [reply shared/js/content-scope/utils.js:146]: We could instead camelcase on injection into the content script but we'd just move the problem I think as the inclusion of the files requires the kabab case.

**Review by sammacbeth — APPROVED** [review]

**Pull request merged**